en/integrations/plugin-packs/procedures/operatingsystems-linux-nrpe3.md

```diff
@@ -0,0 +1,279 @@
+---
+id: operatingsystems-linux-nrpe3
+title: Linux NRPE3
+---
+
+| Current version | Status | Date |
+| :-: | :-: | :-: |
+| 3.1.6 | `STABLE` | Jun 06 2020 |
```

Comment:
This header shouldn't be included anymore. It will be too hard to maintain.

Reply:
Done.